src/bundle/Resources/public/js/scripts/services/notification.service.js

```diff
@@ -0,0 +1,93 @@
+(function(global, doc) {
+    const eZ = (global.eZ = global.eZ || {});
+
+    const NOTIFICATION_INFO_LABEL = 'info';
+    const NOTIFICATION_SUCCESS_LABEL = 'success';
+    const NOTIFICATION_WARNING_LABEL = 'warning';
+    const NOTIFICATION_DANGER_LABEL = 'danger';
+
+    /**
+     * Dispatches notification event
+     *
+     * @function showNotification
+     * @param {Object} detail
+     */
+    const showNotification = (detail) => {
+        const event = new CustomEvent('ez-notify', {
+            detail,
```

Comment:
It can be in the same line  with the `event` variable declaration

Reply:
What's more, you don't even need this variable and make the whole `const showNotification` function declaration in one line.